ghost/core/core/server/services/posts-public/service.js

```diff
@@ -15,8 +17,22 @@ class PostsPublicServiceWrapper {
         if (config.get('hostSettings:postsPublicCache:enabled')) {
             const cache = adapterManager.getAdapter('cache:postsPublic');
             postsCache = new EventAwareCacheWrapper({
-                cache: cache,
-                resetEvents: ['site.changed'],
+                cache,
+                logging,
+                resetEvents: [
+                    'post.deleted',
+                    'post.published',
+                    'post.published.edited',
+                    'post.unpublished',
+                    'page.deleted',
+                    'page.published',
+                    'page.published.edited',
+                    'page.unpublished',
+                    'post.tag.attached',
+                    'post.tag.detached',
+                    'page.tag.attached',
+                    'page.tag.detached'
```

Comment:
Should we also include the events for any includes (authors, tiers) and maybe `tag.edited` as well?

Sort of sucks that they are all stored in the same request body so we can't be more surgical, but I can't think of a way around it unless we fully rethink the approach of caching response bodies.

Reply:
Ah yeh we should definitely clear on those events based on how we are caching the data. Have added:

- `tag.edited`
- `tag.deleted`
- `user.edited` (author)
- `user.deleted` (author)